pl/whats_next/README.md

```diff
@@ -10,16 +10,33 @@ Potem, upewnij się, że śledzisz Django Girls na [Facebooku](http://facebook.c
 
 ### Czy możesz polecić kolejne materiały?
 
-Tak! Najpierw wypróbuj naszą inną książkę o nazwie [Django Girls Tutorial: Extensions](https://tutorial-extensions.djangogirls.org/).
+Tak! Jest *wiele* materiałów online do nauki dowolnych umiejętności programistycznych - może to być całkiem zniechęcające, gdy trzeba wybrać, co zrobić następne, ale my mamy o to zadbałyśmy. Niezależnie od tego czym interesowałaś się zanim przybyłaś na Django Girls i niezależnie od tego co zainteresowało Cię podczas przechodzenia tutorialu, oto kilka wolnych zasobów (lub zasobów z dużymi wolnymi komponentami), których możesz używać aby dostać się tam, gdzie chciałabyś być.
```

Comment:
ale my mamy o to zadbałyśmy -> please rewrite
add commas to the second sentence